google/ggrc-core
# GGRC-4031 Removes ability modify verified and finished date via import

# Issue description

Actual Finish Date / Actual Verified Date contains 'none' for Cycle Task if user removes it via import.

User should not have a possibility to set Actual Finish Date / Actual Verified Date using import at all.

# Steps to test the changes

1. Create any WF (Verification flag - TRUE)
2. Add 1 task to Task Group
3. Activate WF
4. In Active Cycles tab Verify any cycle task: confirm Actual Finish Date / Actual Verified Date contains verified date
5. Export Cycle Task of the Cycle
6. Remove Actual Finish Date / Actual Verified Date and type double dash instead and Import the Cycle task

Result: You are not able to modify Cycle Task this way - the line gets ignored with according warning message.

# Solution description

- Removed 'finished_date' and 'verified_date' aliases from the list of importable fields in cycle task group object task model.

- Moved 'finished_date' and 'verified_date' aliases from ggrc_workflows/converters/handlers.py to ggrc/converters/column_handlers.py. This change was done because these aliases are used by Assessment model which is situated in the main application module.

- Removed code that additionally processed 'Actual Finish Date' and 'Actual Verified Date' for cycle_task_group_object_tasks fields during import.

- Removed help text for CSV fields 'Actual Finish Date' and 'Actual Verified Date'.

- Updated existing integration tests.

- Updated test CSVs according to changes. In particular cases completely removed cycle tasks which contained state 'Finished' and 'Verified'.

- Fixed existing linter issues.

# Sanity checklist




# PR Review checklist
